src/Login.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './Login.css';
+import logo from './Assets/logo.png';
+import mockUsers from '../src/mockusers';
+
+const Login = ({ onLogin }) => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const user = mockUsers.find(
+      (user) => user.username === username && user.password === password
+    );
+    if (user) {
+      onLogin(username, password);
+    } else {
+      alert('Invalid username or password');
+    }
```

Comment:
1. You can get rid of all this code and just invoke the function the way you did. You're already doing all of this in App.js
2. We should clear the form state when we submit so that we don't have values in state that don't need to be there.

Reply:
The alert is a nice catchall, but some more extensive errors on the page could be helpful for the user, i.e. 'please fill out all fields', would be nice to have it rendered to the page rather than an alert

Reply:
yes! your totally right @ Jason. Gonna refactor this.

Agreed fully @ Alec . gonna look into more extensive error handling for the page

Reply:
I removed this redundant code  which still works within the Login component. I also implemented the form state to be cleared after a successful or unsuccessful login.